src/diffusers/loaders.py

```diff
@@ -830,6 +830,11 @@ def load_lora_weights(self, pretrained_model_name_or_path_or_dict: Union[str, Di
         self.load_lora_into_text_encoder(
             state_dict, network_alpha=network_alpha, text_encoder=self.text_encoder, lora_scale=self.lora_scale
         )
+        is_sdxl = any("text_encoder_2" in key for key in state_dict)
+        if is_sdxl:
+            self.load_lora_into_text_encoder(
+                state_dict, network_alpha=network_alpha, text_encoder=self.text_encoder_2, lora_scale=self.lora_scale
+            )
 
     @classmethod
     def lora_state_dict(
```

Comment:
Can we add a test here showing that a two text encoder LoRA can be loaded?

Reply:
Works for you? 

https://github.com/huggingface/diffusers/pull/4097/files#diff-db771746ed12c66b4bcf74a5fe059cfb9c170902d60bbce65886992a9bee34d4R388

Reply:
I thought more about just a quick fast test that loads and unloads a dummy LoRA that is in the format of SD-XL

Reply:
Added in `test_lora_layers.py`.